src/platform/forms/validations.js

```diff
@@ -208,18 +208,6 @@ function isValidRoutingNumber(value) {
   return false;
 }
 
-function getFileError(file) {
-  if (file.errorMessage) {
-    return file.errorMessage;
-  } else if (file.uploading) {
-    return 'Uploading file...';
-  } else if (!file.confirmationCode) {
-    return 'Something went wrong...';
-  }
-
-  return null;
-}
```

Comment:
This is duplicate code (see `forms-system/src/js/validation.js`) that isn't used anywhere else